pkg/gadgets/run/types/metadata.go

```diff
@@ -291,37 +283,37 @@ func getColumnSize(typ btf.Type) uint {
 }
 
 func (m *GadgetMetadata) populateTracers(spec *ebpf.CollectionSpec) error {
-	traceMap := getTracerMapFromeBPF(spec)
-	if traceMap == nil {
-		log.Debug("No trace map found")
+	traceInfo, err := getTracerInfo(spec)
```

Comment:
In most of this function it was used `trace` instead of `tracer` for the info, map, struct, etc:

```suggestion
	tracerInfo, err := getTracerInfo(spec)
```

Reply:
`traceMap` and `traceMapStruct` also need to be fixed.